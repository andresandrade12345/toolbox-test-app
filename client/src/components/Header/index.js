import React from 'react';
import { Container } from 'reactstrap';

const Header = () => {
    return (
        <header>
            <Container>
                <div className="header-content">
                    <div className="header-title text-center">
                        <h1>Toolbox Application</h1>
                    </div>
                </div>
            </Container>
        </header>
    )
};

export default Header;