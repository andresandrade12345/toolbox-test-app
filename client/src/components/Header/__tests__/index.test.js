import React from 'react';
import Header from '../';

describe('test:Header', () => {

    test('renders and matches snapshot', () => {
        const wrapper = shallow(
            <Header />
        );

        return expect(wrapper).toMatchSnapshot();
    });

});