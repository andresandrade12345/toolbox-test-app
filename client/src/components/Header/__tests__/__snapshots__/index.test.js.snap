// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test:Header renders and matches snapshot 1`] = `
<header>
  <Container
    tag="div"
  >
    <div
      className="header-content"
    >
      <div
        className="header-title text-center"
      >
        <h1>
          Toolbox Application
        </h1>
      </div>
    </div>
  </Container>
</header>
`;
